releases/note-source/v14_0.rst

```diff
@@ -0,0 +1,373 @@
+
+:jirab:`DM-6718`
+
+.. _release-v14-0-bug-fixes:
+
+Bug Fixes
+---------
+
+- :ref:`release-v14-0-return-code`
+- :ref:`release-v14-0-version-checking`
+- :ref:`release-v14-0-cmodel-prior`
+- :ref:`release-v14-0-blended-debias`
+- :ref:`release-v14-0-temp-local-bg`
+- :ref:`release-v14-0-peak-cull`
+
+.. _release-v14-0-return-code:
+
+Command line tasks now provide standard Unix return codes
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+When a command line task fails, it will now return a non-0 exit code to the calling shell, in line with standard Unix conventions.
+
+:jirab:`DM-4141`
+
+.. _release-v14-0-version-checking:
+
+Deterministic package version checking
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+The codebase keeps track of which software versions were used to calculate a particular set of outputs as a means of tracking data provenance.
+However, in some circumstances this list was being stored inconsistently, leading to version mismatches being reported where non existed.
+This has now been resolved.
+
+:jirab:`DM-9439`
+
+.. _release-v14-0-cmodel-prior:
+
+Correct CModel priors
+^^^^^^^^^^^^^^^^^^^^^
+
+CModel priors are now weighted correctly relative to the likelihood.
+
+:jirab:`DM-9795`
+
+.. _release-v14-0-blended-debias:
+
+Correct blendedness debiasing calculation
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Resolved a mathematical error.
+
+:jirab:`DM-10237`
+
+.. _release-v14-0-temp-local-bg:
+
+Fixed order of operations when using temporary local backgrounds in detection
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+Resolved an issue which could result in missed isolated-object detections and incorrect ``Footprint``\s for large objects.
+
+:jirab:`DM-10271`
+
+.. _release-v14-0-peak-cull:
+
+Correct number of bands used for peak culling
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+During multi-band coadd processing, peaks may be rejected (or “culled”) if they have been detected in insufficiently many bands.
+A book-keeping error led to the number of bands in which a peak had been detected to be recorded incorrectly, and hence to the wrong peaks being culled.
+This has now been corrected.
+
+:jirab:`DM-11625`
+
+.. _release-v14-0-internal-improvements:
+
+Build and Code Improvements
+---------------------------
+
+- :ref:`release-v14-0-yaml-policy`
+- :ref:`release-v14-0-py3`
+- :ref:`release-v14-0-flaghandler`
+- :ref:`release-v14-0-numpy`
+- :ref:`release-v14-0-auto-bt`
+
+.. _release-v14-0-yaml-policy:
+
+YAML-based policy
+^^^^^^^^^^^^^^^^^
+
+The “policy” files associated with standard stack cameras, which define where and how datasets are persisted by the Butler, were updated to use a new `YAML <http://yaml.org>`_-based syntax.
+
+:jirab:`DM-7363`
+
+.. _release-v14-0-py3:
+
+Full support for Python 3.5
+^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+All of the Science Pipelines code included in this release is tested with both Python 2.7 and Python 3.5.
```

Comment:
We do have that one known py3 problem.

Reply:
Who's tasked with updating the Known issues (e.g. https://pipelines.lsst.io/v/DM-12044/known-issues.html#known-issues)? cc @frossie